lib/mathlib.cpp

```diff
@@ -356,8 +356,6 @@ MathLib::bigint MathLib::characterLiteralToLongNumber(const std::string& str)
     // is implementation-defined.
     // clang and gcc seem to use the following encoding: 'AB' as (('A' << 8) | 'B')
     const std::string& normStr = normalizeCharacterLiteral(str);
-    if (normStr.empty())
-        throw InternalError(0, "Internal Error. MathLib::characterLiteralToLongNumber: Unhandled char constant '" + str + "'.");
```

Comment:
I see the semantics of normalizeCharacterLiteral as "give me a valid literal or throw". Should we encounter crashes in the future, I'd rather we fix normalizeCharacterLiteral than leave them go unnoticed.